v20.2/disaster-recovery.md

```diff
@@ -0,0 +1,337 @@
+---
+title: Disaster Recovery
+summary: Learn how about CockrachDB disaster recovery capabilities and what to do if you encounter an issue.
+toc: true
+---
+
+CockroachDB is built to be [fault-tolerant and to recover automatically](demo-fault-tolerance-and-recovery.html), but sometimes disasters happen. A _disaster_ is any event that puts your cluster at risk, and usually means your cluster is experiencing [hardware failure](#hardware-failure), [data failure](#data-failure), or has [compromised security keys](#compromised-security-keys). Having a disaster recovery plan enables you to recover quickly, while limiting the consequences.
+
+## Hardware failure
+
+When planning to survive hardware failures, these are the minimum replication factors to apply in order to have greater resilience.
+
+{{site.data.alerts.callout_info}}
+Increasing data replication factors may have an impact on [transaction](transactions.html) performance, since quorum levels are increasing.
+{{site.data.alerts.end}}
+
+### Single-region survivability planning
+
+The table below displays the minimum replication factor needed to survive simultaneous hardware failures (e.g., disk, node, availability zone (AZ), etc.) for a single-region CockroachDB cluster:
+
+<table>
+  <thead>
+    <tr>
+      <th>Simultaneous Failure</th>
+      <th>3 nodes <br>(3 AZ/Racks)</th>
```

Comment:
It's not clear the total number of nodes here. I saw rf=5 and 3 nodes and went "huh?".